lib/load.php

```diff
@@ -113,6 +113,7 @@ function gutenberg_is_experiment_enabled( $name ) {
 require __DIR__ . '/experiments-page.php';
 require __DIR__ . '/global-styles.php';
 require __DIR__ . '/query-utils.php';
+require __DIR__ . '/duotone-filter.php';
```

Comment:
Minor: Maybe we can just move the content of this file to the block support file since it's just a private API used in that file. Potentially we could also remove the function and inline it in the "gutenberg_render_duotone_support" like done in the layout support flag.